frontend/src/components/CoursePage.js

```diff
@@ -33,7 +58,7 @@ const TutorPage = () => {
             let coursesuppercased = courses.map(courses => courses.toUpperCase());*/
             let avail = _tutors.at(i).availability;
             console.log(avail);
-            let availuppercased = avail.map(avail => avail.toUpperCase());
+            let availuppercased = avail => avail.toUpperCase();
```

Comment:
This doesn't seem accurate; it's setting it to a lambda. The API should be returning a list here so I think the original is correct.